cpp/src/arrow/compute/kernels/scalar_string.cc

```diff
@@ -4018,6 +4036,13 @@ const FunctionDoc ascii_lower_doc(
      "non-ASCII characters, use \"utf8_lower\" instead."),
     {"strings"});
 
+const FunctionDoc ascii_capitalize_doc(
+    "Capilatize the first character of ASCII input",
+    ("For each string in `strings`, return a capitalized version.\n\n"
+     "This function assumes the input is fully ASCII.  If it may contain\n"
+     "non-ASCII characters, use \"utf8_capitalize\" instead."),
```

Comment:
I don't see utf8_capitalize in this PR. Is this an oversight?

Reply:
Working on it locally, I should have made a draft PR.